pkg/services/featuremgmt/registry.go

```diff
@@ -390,6 +390,11 @@ var (
 			State:       FeatureStateAlpha,
 		},
 		{
+			Name:        "disablePrometheusExemplarSampling",
+			Description: "Disable Prometheus examplar sampling",
+			State:       FeatureStateStable,
+    },
+    {
```

Comment:
Braces are not aligned. Did you update this place by yourself? If so please do not. Check: https://github.com/grafana/grafana/blob/main/pkg/services/featuremgmt/registry.go#L1-L5